tachyons-extra.css

```diff
@@ -83,3 +133,48 @@
 .prewrap {
   white-space: pre-wrap;
 }
+
+.user-select-none, .user-select-none * {
+  user-select: none;
+}
+
+.transition {
```

Comment:
we could also think of more specific transitions and/or with different timing

Reply:
`.transition--fast` and `.transition--slow`?

Reply:
I was thinking something like `.transition-all-1` `.transition-all-2` and `.transition-width-1` `.transition-width-2` etc.. which is more "tachyons style" but maybe it also makes sense to keep those more explicit in this case. 
Anyway I think it's important to differentiate the type of transition too (maybe we only need to animate a specific css property and no others)

Reply:
I don't think specific transitions for every property should be in `tachyons-extra`, just create the transition with the property you need when you need it. The `transition` is really for testing, plug and play. You shouldn't animate the width property anyway.

Reply:
🤔not really sure why animating the width property should be avoided. 
If the `.transition` class is just for testing purposes then I don't think it makes sense to include it in the bundle, you can just put it as an inline style attribute for testing. 
I think having a predefined set of transition classes for most used properties can actually be worth it. And then of course you can always create a custom one if you need it.
Curious what are @accurat/developers thoughts about this.

Reply:
Better discuss this here anyway: https://github.com/accurat/tachyons-extra/issues/4